### PR TITLE
Remove decimal commas

### PR DESCRIPTION
**Pull request by jimmay5469**
Resolves: #165 

Old:
`1,234.003,102,702,000,000,000`

New:
`1,234.003102702`

# Changelog

## Enhancements
- Use custom format for `Cldr.Number.to_string!` to remove commas in decimal and trailing zeroes
- Always show at least 2 decimal places for USD values

## Additions
- Add function to create USD structs from decimal representations

### CODE REVIEW

File: apps/explorer_web/lib/explorer_web/exchange_rates/usd.ex [hunk 1]
```diff
@@ -17,6 +17,12 @@ defmodule ExplorerWeb.ExchangeRates.USD do
   alias Explorer.Chain.Wei
   alias Explorer.ExchangeRates.Token
 
+  def from(nil), do: null()
+
+  def from(%Decimal{} = usd_decimal) do
+    %__MODULE__{value: usd_decimal}
```
Comment (KronicDeth) [inline apps/explorer_web/lib/explorer_web/exchange_rates/usd.ex]: This line is uncovered according to https://coveralls.io/builds/17126419/source?filename=apps%2Fexplorer_web%2Flib%2Fexplorer_web%2Fexchange_rates%2Fusd.ex#L1.  Are you sure this is needed OR is a test missing that uses it?

File: apps/explorer_web/lib/explorer_web/views/currency_helpers.ex [hunk 2]
```diff
@@ -28,7 +28,7 @@ defmodule ExplorerWeb.CurrencyHelpers do
   def format_usd_value(%USD{value: nil}), do: nil
 
   def format_usd_value(%USD{value: value}) do
-    case Number.to_string(value, format: "#,##0.##################") do
+    case Number.to_string(value, format: "#,##0.00################") do
```
Comment (KronicDeth) [inline apps/explorer_web/lib/explorer_web/views/currency_helpers.ex]: Neat, I didn't know `00` vs `##` made sure the `00` showed up.

File: apps/explorer_web/lib/explorer_web/exchange_rates/usd.ex (hunk 1 above)
Comment (jimmay5469) [reply apps/explorer_web/lib/explorer_web/exchange_rates/usd.ex]: I believe it just needs a test.

**Comment by KronicDeth** [comment]
https://coveralls.io/builds/17133098/source?filename=apps%2Fexplorer_web%2Flib%2Fexplorer_web%2Fviews%2Fchain_view.ex#L19 is show as uncovered after the latest changes, which would seem to indicate we're missing a test for non-nil USD value for the exchange rate.
